Localization/cubature_kalman_filter/cubature_kalman_filter.py

```diff
@@ -0,0 +1,234 @@
+"""
+implementation of cubature kalman filter using CTRV motion model
+
+# author: Raghuram Shankar
+# based on PythonRobotics by Atsushi Sakai
+
+"""
+
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.linalg import sqrtm
+
+# initalize global variables
```

Comment:
initialize?

Reply:
Initialize means to declare and assign values to variables.

Reply:
I mean this is typo `initalize` -> `initialize`

Reply:
Oh, I didn't catch that before!